docs/source/advanced/link-prediction.rst

```diff
@@ -328,3 +329,76 @@ For example, the file storing hard negatives should look like the following:
     "src_100"| "dst_41"| "dst0;dst_2"
 
 GraphStorm will automatically translate the Raw Node IDs of hard negatives into Partition Node IDs in a DistDGL graph.
+
+.. _link-prediction-evaluation-metrics:
+
+Link Prediction Metrics
+-----------------------
+
+GraphStorm supports several metrics for link prediction, to give a well-rounded
+view of model performance.
+In general, link prediction evaluation happens by constructing a set of negative
+edges with one of the sampling methods described above, and including one positive
```

Comment:
When doing testing, usually we uniformly sample negative edges. (We do not use joint negative or in-batch negative.

Reply:
What's our default? The docs say:

> eval_negative_sampler: The negative sampler used for link prediction testing and evaluation. Built-in samplers include uniform, joint, localuniform, all_etype_uniform and all_etype_joint.

        Yaml: eval_negative_sampler: uniform

        Argument: --eval-negative-sampler joint

        **Default value: joint**

Reply:
Oh. We use joint by default.